bofire/domain/features.py

```diff
@@ -1403,11 +1425,22 @@ def get_categorical_combinations(
         features = [
             f
             for f in self.get(includes=include, excludes=exclude)
-            if isinstance(f, CategoricalInput) and not f.is_fixed()
+            if (isinstance(f, CategoricalInput) and not f.is_fixed())
         ]
         list_of_lists = [
             [(f.key, cat) for cat in f.get_allowed_categories()] for f in features
         ]
+
```

Comment:
I do not understand why you need this, this should automatically work or when you include the `DiscreteInput` in the includes, or not?

Reply:
Ah I undertand it, what happens if you have discrete in the exclude?